nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -56,6 +56,12 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
 - Invidious has changed its default database username from `kemal` to `invidious`. Setups involving an externally provisioned database (i.e. `services.invidious.database.createLocally == false`) should adjust their configuration accordingly. The old `kemal` user will not be removed automatically even when the database is provisioned automatically.(https://github.com/NixOS/nixpkgs/pull/265857)
 
+- `services.frp.settings` now generates the frp configuration file in TOML format as recommended by upstream, instead of the legacy INI format. This has also introduced other changes in the configutation file structure and options.
+  - The `setttings.common` section in the configuration is no longer valid and all the options form inside it now goes directly under `settings`.
```

Comment:
A few typos. Also consider adding a link on `recommended by upstream`, as people might be interested in these changes.

```suggestion
- `services.frp.settings` now generates the frp configuration file in TOML format as recommended by upstream, instead of the legacy INI format. This has also introduced other changes in the configuration file structure and options.
  - The `settings.common` section in the configuration is no longer valid and all the options form inside it now goes directly under `settings`.
```